### PR TITLE
new blog post pitting Apache Kafka and the OpenTelemetry collector in…

### PR DESCRIPTION
**Pull request by serverless-mom**
a knockdown brawl (not really)

### CODE REVIEW

**Review by serverless-mom — COMMENTED** [review]
There's some weird artifact here with .png formatting here. File is still there

File: blog/2023-10-12-apache-kafka-and-opentelemetry-collector-for-better-burst-handling.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+The result will be errors raised on the app when it fails to reach the collector for every single request.
+
+Again, the whole benefit of the collector is that it should be able to cache, batch, and compress data for sending, no matter how high-frequency its data ingest.
+
+## Multi-collector architecture
+
+<figure data-zoomable align='center'>
+    <img src="/img/blog/2023/10/c2c-3.webp" alt="diagram of a multi-collector architecture"/>
+    <figcaption><i>A collector, B, running very close to the service being instrumented could collect data reliably, and batch it before sending to second, central collector, C. The C collector could gather data from multiple other ‘front-line’ collectors before sending to a data backend. </i></figcaption>
```
Comment (ankit01-oss) [inline blog/2023-10-12-apache-kafka-and-opentelemetry-collector-for-better-burst-handling.md]: @serverless-mom I don't see collector `B` and `C` marked in the picture. Is there an `A` collector too?

File: blog/2023-10-12-apache-kafka-and-opentelemetry-collector-for-better-burst-handling.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+1. Ensuring data collection during database outages - even reliable databases fail, and Kafka can ingest and store data during the outage. When the DB is up again, the consumer can start taking in data again
+2. Handling traffic bursts - observability data can spike during a usage spike, and if you’re doing deep tracing the spike can be even larger in scale than the increase in traffic. If you scale your database to handle this spike without any queueing, the DB will be over-provisioned for normal traffic. A queue will buffer the data spike so that the database can handle it when it’s ready.
+
+<figure data-zoomable align='center'>
+    <img src="/img/blog/2023/10/c2c-3.webp" alt="diagram with multiple collectors with an added Kafka queue"/>
+    <figcaption><i>in this new version, a Kafka queue receives data from collectors near the edge. Services could also publish directly to Kafka. Collector C reads the data from the queue, using the OTel Kafka Receiver</i></figcaption>
```
Comment (ankit01-oss) [inline blog/2023-10-12-apache-kafka-and-opentelemetry-collector-for-better-burst-handling.md]: This picture looks the same as above. Can we have kafka represented in the diagram too? 

Also labeling of collectors is missing.

File: blog/2023-10-12-apache-kafka-and-opentelemetry-collector-for-better-burst-handling.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
+---
+title: Maximizing Scalability - Apache Kafka and OpenTelemetry
```
Comment (ankit01-oss) [inline blog/2023-10-12-apache-kafka-and-opentelemetry-collector-for-better-burst-handling.md:2]: Can we make the title more use-case oriented? Not sure about how we can - just want to let users have more idea on what the article has to offer. Or just make it more simpler like - 

`Using Apache Kafka and OpenTelemetry Collector to maximize scalibility`.

**Review by ankit01-oss — COMMENTED** [review]
have left some comments @serverless-mom. 

Also, you can run it through Grammarly once. Noticed some punctuation marks missing at places.

**Pull request merged**